docs/tutorials/user-transpiled-circuits.ipynb

```diff
@@ -0,0 +1,2511 @@
+{
```

Comment:
<p>Can we add link to the quick start guide of IBM Cloud to explain how to set up the service / save account?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-ibm-runtime/pull/231/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>I linked to this page (https://cloud.ibm.com/docs/quantum-computing?topic=quantum-computing-quickstart) but I am not sure if this is appropriate because it's on IBM Cloud. I think the idea is to link to the Getting Started page on the doc but we don't have that page yet.</p>

Reply:
Ok cool, I'll wait until that's available and add it then

Reply:
I'm going to leave this for now as the page isn't available yet, but I'll make a follow-up when it is